databrowser/src/main.ts

```diff
@@ -13,6 +14,7 @@ app.use(router);
 
 // Add axios and provide it as injectable property (see https://www.npmjs.com/package/vue-axios)
 app.use(VueAxios, axios);
+app.use(store);
```

Comment:
Putting this declaration here splits the axios config into two pieces. This is not a big deal, we can reorder it in one of the next commits in order to be able to apply and close this PR.